### PR TITLE
Refactoring 2 12 24

### PR DESCRIPTION
**Pull request by azheng1991**
Removed unnecessary app.py files, updated README text

### CODE REVIEW

**Comment by azheng1991** [comment]
@all-contributors please add @azheng1991 for code contributions

File: README.md [hunk 1]
```diff
@@ -11,12 +11,9 @@ This repository contains a pipeline from taking bills from Massachusetts legisla
 * [EDA](https://github.com/vynpt/ml-maple-bill-summarization/tree/dev/EDA): the notebook ```eda.ipynb``` includes our work from scraping data that takes bills from MAPLE Swagger API, creating a dataframe to clean and process data, making visualizations to analyze data and explore characteristics of the dataset.
   
 * [demoapp](https://github.com/vynpt/ml-maple-bill-summarization/tree/dev/demoapp):   
-  ```app.py```: contains the codes of the LLM service we used and the wepapp we made using Streamlit. The webapp allows user to search for all bills.  
-  ```demo_app_with_12bills.py```: we test on top 12 bills from MAPLE website. We extract information from [Massachusetts General Law](https://malegislature.gov/Laws/GeneralLaws) to add context for the summaries of these bills. MGL sections text was scraped using extract_mgl_sections.py. Contains code to generate bill categories and tags
```
Comment (nesanders) [inline README.md:15]: The `demo_app_with_12bills` text should not be deleted - that app is still in use!

File: README.md [hunk 2]
```diff
@@ -11,12 +11,9 @@ This repository contains a pipeline from taking bills from Massachusetts legisla
...
-
   ```demo_app.py```: contains the codes of the LLM - OpenAI service and webapp made using Streamlit. The webapp allows user to search for all bills. MGL sections text is extracted for all but ~1300 bills and is available for in 'Combined_MGL' column in all_bills_with_mgl.pq file (currently hosted on google drive due to it's large size).
 
-  Both demo_app.py and demo_app_with_12bills.py generate bill category and tags (based on a given list) and summarize the bill text for 12 bills and all bills, respectively. We currently use vectorstore to split the MGL document into chunks for vectorstore storage and embeddings before injection into the prompt. However, we would like to test injecting the MGL sections directly into the prompt without using vectorstores as the operations through vectorstores are fuzzy (rely on similarity search). 
+  We currently use vectorstore to split the MGL document into chunks for vectorstore storage and embeddings before injection into the prompt. However, we would like to test injecting the MGL sections directly into the prompt without using vectorstores as the operations through vectorstores are fuzzy (rely on similarity search).
```
Comment (nesanders) [inline README.md:19]: I think the text you removed here actually still stands; it should be kept.

**Review by nesanders — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (azheng1991) [reply README.md:15]: fixed

File: README.md (hunk 2 above)
Comment (azheng1991) [reply README.md:19]: fixed

**Comment by azheng1991** [comment]
I also added .ipynb_checkpoints/ to the gitignore file because every time I was trying to add files to the staging area, it was also adding the checkpoint files, which I must have created when I was looking into some of the dataframe indexing stuff we talked about.

File: README.md (hunk 1 above)
Comment (nesanders) [reply README.md:15]: @azheng1991 are you sure you pushed this fix? I still see the text as being deleted for 12bills.

**Review by nesanders — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (azheng1991) [reply README.md:15]: I believe so...the changes I made are in the commit history and showing up on the branch at least:
image

Comment (nesanders) [reply README.md:15]: Got it, I see it. Thanks!

**Review by nesanders — APPROVED** [review]

**Pull request merged**